### DISCUSSION TITLE
Problem in running 02-QbC code in example

### DISCUSSION THREAD

**Discussion by semoday** [MarsalekGroup/aml#8] [Q&A] [answered]
Hello,
I am trying to run the example simulation on zundel. 
Firstly, I did generate the reference trajectory using CP2K (01-AIMD-zundel-DFTB2)
However when I run the run-QbC.py code (02-QbC)
I get an error that says that

TypeError: 'NoneType' object is not iterable

Iteration did not start and nothing was created in iteration-001/train-000 directory. 
Can you help me solve this problem ?

**Comment by cschran** [comment, marked as answer]
Thanks a lot for making us aware. This was a bug introduced when porting this feature from the developers repo.
Sorry for the inconvenience.  The latest commit fixes this issue. The example should now run without any problems.

**Reply by semoday** [reply]
Thank you for your quick answer!
The example now runs without any problems.
